debian/control

```diff
@@ -18,7 +18,7 @@ Build-Depends: automake,
                dh-exec,
                dh-python,
                dh-systemd,
-# Jaeger       flex,
+               flex  <pkg.ceph.jaeger>,
```

Comment:
could you revise the commit message ?

1. the title is scattered into 3 lines. it should read
  > debian, install-deps.sh: use Build-Profiles to optionally build with jaeger
2. the body of commit message is difficult to parse.